content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -260,7 +260,52 @@ unifiedControlPlaneImage: ""
 ### Adding kube-proxy parameters {#kube-proxy}
 
 For information about kube-proxy parameters in the MasterConfiguration see:
-- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig/v1alpha1#KubeProxyConfiguration)
+- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/config#KubeProxyConfiguration)
+
+For information about enabling IPVS mode with kubeadm see:
+- [IPVS](https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/README.md)
+
+In Kubernetes 1.11 and later, Kube-proxy will run in ipvs mode by default in a cluster deployed by kubeadm.
```

Comment:
please remove the details from lines `267` to `308`
as mentioned on slack this is not needed as this document covers it:
https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/README.md

Reply:
ok, done.